LoopKitUI/View Controllers/ChartsTableViewController.swift

```diff
@@ -0,0 +1,151 @@
+//
+//  ChartsTableViewController.swift
+//  LoopKitUI
+//
+//  Copyright © 2017 LoopKit Authors. All rights reserved.
+//
+
+import UIKit
+import HealthKit
+
+/// Abstract class providing boilerplate setup for chart-based table view controllers
+open class ChartsTableViewController: UITableViewController, UIGestureRecognizerDelegate {
+
+    public var preferredGlucoseUnit: HKUnit?
```

Comment:
This was added by me to shift the responsibility of unit updates to the view containing the chart